### ISSUE TITLE
Cropmarks package error

### ISSUE THREAD

**Issue by mnjames** [sile-typesetter/sile#1680]
I believe this line is deprecated
https://github.com/sile-typesetter/sile/blob/2f9e06d1cd63ae17a3be39c39c74bd5161308701/packages/cropmarks/init.lua#L66

and should now say 
`local info = SILE.masterFilename .. " - " .. self.class.packages.date:date("%x %X") .. " -  " .. outcounter`

(Sorry, writing up this issue was quicker than forking the project and creating a pull request. And I'm not 100% sure my new code is correct.)

**Comment by Omikhleia** [comment]
See also #1285 and #1470 - The proposed change there at the time was:
```
 local info = SILE.masterFilename .. " - " .. self.class:date({ format = "%x %X" }) .. " - " .. outcounter
```
Still seems valid to me that the function argument is expected to be an object, not a string. (Regardless of the deprecation).

**Comment by mnjames** [comment]
I'm assuming this issue can be closed, since the edits at #1470 will likely make anything I've noted here irrelevant.
